src/shared/api/urls.ts

```diff
@@ -134,7 +134,7 @@ export function getVirtualStudyServiceUrl() {
 }
 
 export function getSessionServiceUrl() {
-    return buildCBioPortalAPIUrl("api-legacy/proxy/session/");
+    return buildCBioPortalAPIUrl("api-legacy/proxy/session/main_session");
```

Comment:
is this a desired change?

Reply:
yes. it was actually already done elsewhere